### PR TITLE
Add Begin Wallet Pool

### PR DESCRIPTION
**Pull request by francisluz**
### Commitment

By submitting this PR I agree that I will fight for the full decentralization of Cardano by only running a single pool and reporting members that are breaking the rules by running multiple pools.

#### PR Checklist

I have:
- [x] Added a telegram handle of an **individual** and not that of a group or channel.
- [x] Added a social media account with activity.
- [x] Removed any `@` characters from Telegram, Instagram or Twitter handles.
- [x] Provided my operator name as my real full name, my real first name, or I have left that field blank.
- [x] Included all social media fields in the PR even if they are blank (this will fail validation if not done).

### CODE REVIEW

File: registry.json [hunk 1]
```diff
@@ -151,6 +151,20 @@
...
+    "poolId": "b59bf139701b8cdae5c997225f438db7603ef1d9fd9a1f62dc5821eb",
+    "social": {
+      "website": "https://begin.is"
+      "twitter": "BeginWallet",
+      "facebook": "beginwallet",
+      "instagram": "beginwallet",
+      "youtube": "beginwallet"
+    },
+    "telegram": "francisluz",
+    "operatorName": "B58 Finance",
```
Comment (Bez625) [inline registry.json:165]: Please change this to be the operators real name or leave blank

**Review by Bez625 — CHANGES_REQUESTED** [review]

File: registry.json (hunk 1 above)
Comment (francisluz) [reply registry.json]: Thanks @Bez625, change done.

File: registry.json [hunk 2]
```diff
@@ -151,6 +151,20 @@
     "operatorName": "CONRAD",
     "github": "conraddit"
   },
+  {
+    "ticker": "BEGIN",
+    "poolId": "b59bf139701b8cdae5c997225f438db7603ef1d9fd9a1f62dc5821eb",
+    "social": {
+      "website": "https://begin.is"
```
Comment (Bez625) [inline registry.json:158]:
```suggestion
      "website": "https://begin.is",
```

Comment (Bez625) [reply registry.json:158]: I cannot update this, but you missed a comma in the json and validation failed. Please fix

**Review by Bez625 — CHANGES_REQUESTED** [review]
json is invalid, please fix

**Comment by Bez625** [comment]
Please address the comments or I will look to close soon

**Comment by francisluz** [comment]
Sorry for the delay, json fixed.

**Pull request merged**